Documentation/networking/examples-bridge.md

```diff
@@ -0,0 +1,46 @@
+## Using NAT with bridge
+
+The bridge plugin can be configured to create a separate network on the host that will be NAT'ed similar to the default network.
```

Comment:
"bridge plugin" can be a link to:

https://github.com/appc/cni/blob/master/Documentation/bridge.md